templates/api/knowledge-panels/health/ingredients/ingredient.tt.json

```diff
@@ -0,0 +1,37 @@
+{
+    "level": "info",
+    "topics": [
+        "health"
+    ],
+    "size": "small",
+    [% IF panel.ingredient_description OR panel.wikipedia_abstract %]
+    "expanded": false,
+    [% ELSE %]
+    "expanded": false,
+    [% END %]
+    "title_element": {
+        // Note: the app displays line feeds as line feeds...
+        "title": `        [% dash = "-" %]        [% dash.repeat(panel.level) %]        [% display_taxonomy_tag_name("ingredients", panel.ingredient_id) %]        [% IF panel.ingredient.percent.defined %]            ([% round(panel.ingredient.percent) %]%)        [% ELSIF panel.ingredient.percent_estimate.defined %]            ([% round(panel.ingredient.percent_estimate) %]% [% lang("estimate") %])        [% END %]        `,
```

Comment:
Can't we eventually deport this computation in the ingredient panel creation ?

Reply:
Well the idea of using Template::Toolkit was that people could change the layout / design decisions from the panels, without having to change the Perl code. So that's why I try to put things that are more design things in the template. Here I tried to show the sub-ingredients, and I'm just pushing dashes in front, which is not very clear (but it's something..). There may be other better ways to do that.